### PR TITLE
chore(CPP): integrate tokens balance in UI POC

### PR DESCRIPTION
**Pull request by stefandunca**
Beware, it is rebased on top of https://github.com/status-im/status-desktop/pull/6618. Only two commits on top are new.

Second commit

- [x] ~~Network and tokens static config to define/fetch the default networks, tokens and currency to be mixed for the final balance~~
- [x] ~~BalanceService to pull, filter, aggregate and cache balances~~
- [x] ~~Networks controller and model~~
- [x] AccountAssetController with hardcoded values and model

First commit

- [x] Implements add watch-only account to UI for quick balance test and visualization

Closes: https://github.com/status-im/status-desktop/issues/6321

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

**Review by MishkaRogachev — APPROVED** [review]
Great work!

File: libs/Wallet/qml/Status/Wallet/AssetsPanel.qml [hunk 1]
```diff
@@ -109,32 +123,72 @@ Item {
...
+                component NewAccountEntry: ItemDelegate {
+                    required property string title
+                    required property Component sourceComponent
+
+                    text: title
+                    width: accountTypeComboBox.width
+                }
+                model: ObjectModel {
+                    NewAccountEntry {
+                        title: "New Account"
```
Comment (caybro) [inline libs/Wallet/qml/Status/Wallet/AssetsPanel.qml:154]: Not totally necessary at this point, but `qsTr()`. These "temp" things tend to get forgotten in the end :)

File: libs/Wallet/src/AccountAssetsController.cpp [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+#include <QtConcurrent>
+
+namespace WalletGo = Status::StatusGo::Wallet;
+
+namespace Status::Wallet {
+
+AccountAssetsController::AccountAssetsController(WalletAccount* address, QObject* parent)
+    : QObject(parent)
+    , m_address(address)
+    , m_enabledTokens({"SNT", "ETH", "STT", "DAI"})
```
Comment (caybro) [inline libs/Wallet/src/AccountAssetsController.cpp:18]: (some) string literals?

**Review by caybro — APPROVED** [review]
Awesome work

Minor suggestions inline

**Pull request merged**